src/store/conversationsStore.js

```diff
@@ -137,6 +137,26 @@ const getters = {
 	 */
 	conversation: state => token => state.conversations[token],
 	dummyConversation: state => Object.assign({}, DUMMY_CONVERSATION),
+	isModerator: (state, getters) => token => {
+		const conversation = getters.conversation(token)
+		return conversation?.participantType === PARTICIPANT.TYPE.OWNER
+				|| conversation?.participantType === PARTICIPANT.TYPE.MODERATOR
+				|| conversation?.participantType === PARTICIPANT.TYPE.GUEST_MODERATOR
+	},
+	isModeratorOrUser: (state, getters) => token => {
+		const conversation = getters.conversation(token)
+		return !conversation?.isDummyConversation
+			&& (getters.isModerator(token)
+				|| conversation?.participantType === PARTICIPANT.TYPE.USER
+				|| conversation?.participantType === PARTICIPANT.TYPE.USER_SELF_JOINED)
+	},
+	isInLobby: (state, getters) => token => {
+		const conversation = getters.conversation(token)
+		return conversation
+				&& conversation.lobbyState === WEBINAR.LOBBY.NON_MODERATORS
+				&& !getters.isModerator(token)
+				&& (conversation.permissions & PARTICIPANT.PERMISSIONS.LOBBY_IGNORE) === 0
+	},
```

Comment:
These getters are always used with only one token value — the value from the store: `this.$store.getters.getToken()`.

In the previous solution with the composable in this PR the token also was got from `store.getters.getToken()`

We may directly the the value from the store and get rid of computing a function as a value instead of the value itself. Returning a function in computed leads to losing caching of the value so than each component instance runs this function again.

Reply:
> We may directly the the value from the store and get rid of computing a function as a value instead of the value itself.

I am not sure if you meant this 
```
isInLobby() {
			return this.$store.getters.isInLobby
		},
```

instead of this 
```
isInLobby() {
			return this.$store.getters.isInLobby()
		},
```

The first approach is function reference so I think it is not what you meant by accessing the value itself ? 
Anyway, Vuex has its caching advantage too. So, despite that computed instances are constantly calling the computed function and executing it,  the value will be returned instantly as it is cached by Vuex.

Reply:
I guess he meant `context.getters.conversation(context.rootGetters.getToken)` without passing it

Reply:
> The first approach is function reference so I think it is not what you meant by accessing the value itself ?

I meant, that it should not be a function. Getter in Vuex is the same as a computed in components. $store.getters.foo is a function only, when the getter function computes a new function as a value.

> Anyway, Vuex has its caching advantage too. So, despite that computed instances are constantly calling the computed function and executing it, the value will be returned instantly as it is cached by Vuex.

Yes, but when we use a getter to compute a new getter function, the result of the computing and the cached value is a new function, not the final Boolean value. Then each component instance, that uses this getter, then calls this function to compute the value and cache it instances, not the store. Also we cannot see the value in the Vuex devtools.